src/templates/submission/start_proposal.html

```diff
@@ -6,146 +6,194 @@
+                                {% bootstrap_field field %}
+                        {% endif %}
+                    </div>
+                    {% endfor %}
+                </div>
+                {% endif %}
+
+                <br>
+                <label for="proposal-type">Proposal Type</label>
+                <select class="form-control" id="proposal-type" name="proposal-type">
+                    <option value='monograph'>Monograph</option>
+                    <option value='edited_volume'>Edited Volume</option>
+                </select>
+                <br>
+            </div>
+            {% if revise%}
+                <div class="form-group">
+                    <label class="control-label" for="id_note_to_author">Email to Editor</label>
+                    <textarea class="form-control" cols="40" id="email_text" name="email_text" rows="10" title="">
+                        {{ email_text }}
+                    </textarea>
+                </div>
+            {% endif %}
+            {% buttons %}
+                <div class="dash-container">
+                    <button name="book_submit" id="book_submit" type="submit"
+                            formnovalidate class="btn btn-other pull-right"
+                            style="width:120px">
+                        Submit
+                    </button>
+                    <button name="incomplete" name="incomplete" type="submit"
+                            formnovalidate class="btn btn-default pull-left"
+                            style="width:120px">
+                        Save
+                    </button>
+                </div>
+            {% endbuttons %}
     </form>
     <br>
     <br>
     <br>
-        {% if proposal.requestor %} 
-   <table class="table table-bordered">
-    <thead>
-        <tr>
-          <th>Contact Editor</th>
-        </tr>
-    </thead>
-    <tbody>
-    <tr>
-      <td>{{ proposal.requestor.username }} <sup> <a href="core/email.html" onclick="return popitup('/email/proposal/{{proposal.id}}/user/{{proposal.requestor.id}}/','proposal_{{proposal.requestor.id}}')"class="fa fa-envelope-o btn btn-xs btn-default"></a></sup></td>
-    </tr>
-  </tbody>
-  </table>
-  {% else %}
-          <table class="table table-bordered">
-          <thead>
-              <tr>
+    {% if proposal.requestor %}
+        <table class="table table-bordered">
+            <thead>
+            <tr>
+                <th>Contact Editor</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>{{ proposal.requestor.username }}
+                    <sup>
+                        <a href="core/email.html"
+                           onclick="return popitup('/email/proposal/{{proposal.id}}/user/{{proposal.requestor.id}}/','proposal_{{proposal.requestor.id}}')"
+                           class="fa fa-envelope-o btn btn-xs btn-default">
+                        </a>
+                    </sup>
```

Comment:
Is this repeated from the previous file? It should go into its own template. [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).